pylint/lint.py

```diff
@@ -1127,10 +1127,8 @@ def get_ast(self, filepath, modname, data=None):
         try:
             if data is None:
                 return MANAGER.ast_from_file(filepath, modname, source=True)
-
-            cached = MANAGER.astroid_cache.get(modname)
-            if cached and cached.file == filepath:
-                return cached
+            # TODO refactor: Add a new method similar to ast_from_file to the
+            # astroid.manager.AstroidManager class.
             return AstroidBuilder(MANAGER).string_build(data, modname, filepath)
```

Comment:
Note that this is the only place in pylint, where we use the AstroidBuilder directly.

Reply:
Do the travis-ci tests fail because I've introduced a TODO comment, seriously?

Reply:
Yes, seriously. If this TODO makes sense to be an issue, please create one instead, otherwise it's going to rot in the codebase for no reason.

Reply:
Totally makes sense. Thx for the ansewer. I've just created https://github.com/PyCQA/astroid/issues/725